src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/Publisher.java

```diff
@@ -55,7 +56,11 @@ public Publisher(int evalIntervalSeconds, Collator collator) {
 
   @Override
   public EmptyFlowUnit operate() {
-    // TODO: Need to add dampening, avoidance, state persistence etc.
+    return new EmptyFlowUnit(Instant.now().toEpochMilli());
+  }
+
+  public void compute(FlowUnitOperationArgWrapper args) {
+    // TODO: Need to add dampening, avoidance etc.
     Decision decision = collator.getFlowUnits().get(0);
```

Comment:
Maybe we want to verify that the size is 1 ?

Reply:
Added a check.